MC-10904 : Corrected the API to have the proper parameters (#115)
cloudops/cloudmc-api-docs: source/includes/azure/_networks.md

Retrieve a list of all virtual networks in an [environment](#administration-environments)

Optional query parameters | &nbsp;
---------- | -----

> `subnetList`<br/>*boolean* | If it is set to true, it will return the subnet information as well in the list. 

Reviewer: Small suggestion?
```suggestion
`subnetList`<br/>*boolean* | If set to true, it will also include the subnet information in the list. 
```
